### ISSUE TITLE
Branches tags and versions

### ISSUE THREAD

**Issue by BastiaanOlij** [GodotVR/godot_openxr#81]
I believe it's time we start focusing on release `1.1.0` and merge our `android-dev` branch into master.

Official versions are managed through tags, so we currently have `1.0.0`, `1.0.1`, `1.0.2` and `1.0.3` releases tagged. We're also using tags to mark alpha and beta builds.

Our `master` branch is where we merge our development work into and is thus our leading branch. Once we merge our `android-dev` branch this branch effectively moves us up to `1.1.x` territory.

In order to be able to do `1.0.x` patch releases we will therefor create a `1.0.x` branch.

PRs should be submitted against the master branch unless it is a fix only applicable for an older version. If we want the fix to go against both master and an older version we add the tag `cherry-pick-1.0.x`. Once a PR is approved and ready to merge into master we'll create a PR for the cherry pick as well and remove the cherry pick tag to let us know we've handled it.

We will also soon start work on porting this plugin to Godot 4 proper, we'll like create a `godot4-dev` branch to manage this work. Once merged into master that will move our master forward to `2.0.x`
